spiffe-helper.yaml

```diff
@@ -0,0 +1,42 @@
+package:
+  name: spiffe-helper
+  version: 0.9.0
+  epoch: 0
+  description: A helper utility for SPIFFE (Secure Production Identity Framework For Everyone) operations.
+  copyright:
+    - license: Apache-2.0
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/spiffe/spiffe-helper.git
+      tag: v${{package.version}}
+
+  - uses: go/build
+    with:
+      packages: "./cmd/spiffe-helper"
+      output: spiffe-helper
+
+subpackages:
+  - name: spiffe-helper-compat
+    description: Compatibility package for spiffe-helper
+    pipeline:
+      - runs: |
+          mkdir -p ${{targets.subpkgdir}}/
+          ln -sf /usr/bin/spiffe-helper "${{targets.subpkgdir}}"/spiffe-helper
+
+update:
+  enabled: true
+  github:
+    identifier: spiffe/spiffe-helper
+    strip-prefix: v
+
+test:
+  pipeline:
+    - name: "Check symlinks"
+      runs: |
+        test -f /usr/bin/spiffe-helper
```

Comment:
this test will go in the compat subpackge as per Max's comment and we can use `stat /spiffe-helper` here because this will print more verbose output when we run `make test/spiffe-helper`.